docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_proxy/_index.md

```diff
@@ -0,0 +1,132 @@
+
+As mentioned, this deployment will leverage ARM templates. You will deploy a single ARM template at resource group scope.
+
+- Before deploying the ARM template, login to Azure using AZ CLI with the ```az login``` command.
+
+- The deployment will use an ARM template parameters file to customize your environment. Before initiating the deployment, edit the [*azuredeploy.parameters.json*](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/proxy/azuredeploy.parameters.json) file located in your local cloned repository folder. Example parameters files is located [here](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/proxy/azuredeploy.example.parameters.json).
+
+- To deploy the ARM template, navigate to the local cloned [deployment folder](https://github.com/microsoft/azure_arc/tree/main/azure_arc_servers_jumpstart/proxy) and run the below command, start deploying resources:
+
+    ```shell
+    az group create --name <Name of the Azure resource group> --location <Azure region> --tags "Project=jumpstart_azure_arc_servers"
+    az deployment group create \
+    --location <Azure Region Location> \
+    --resource-group <Resource Group Name> \
+    --name <Deployment Name> \
+    --template-file <The *azuredeploy.json* template file location> \
+    --parameters <The *azuredeploy.parameters.json* parameters file location>
+    ```
+
+    > **NOTE: make sure that you are using the same Azure resource group name as the one you’ve just used in the azuredeploy.parameters.json file**
+
+    For example:
+
+    ```shell
+    az group create --name Arc-Proxy-Demo --location <Azure region> --tags "Project=jumpstart_azure_arc_servers" 
+    az deployment group create \
+    --location eastus \
+    --resource-group Arc-Proxy-Demo \
+    --name proxy \
+    --template-uri https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_servers_jumpstart/azure/proxy/azuredeploy.json \
+    --parameters azuredeploy.example.parameters.json
+    ```
+
+- Verify the resources are created on the Azure Portal on the resource group:
+
+    ![Resources created on resource group](./01.png)
+
+## Linux Login & Post Deployment
+
+- Now that the resources are created, it is time to connect to the client's VM.
+
+- At first login, as mentioned in the "Automation Flow" section, a logon script will get executed. This script was created as part of the automated deployment process.
+
+- Let the script to run its course and **do not close** the shell session.
+
+    > **NOTE: The script run time is ~1-2min long.**
+
+    ![Screenshot script output](./02.png)
+
+- Upon successful run, a new Azure Arc-enabled server will be added to the resource group.
+
+  ![Screenshot Azure Arc-enabled server on resource group](./03.png)
+
+## Azure Arc-enabled server Proxy connectivity
+
+To make sure that your Azure Arc-enabled server is using Private Link for its connection. Use your Azure Bastion session to run the command below:
```

Comment:
```suggestion
To make sure that your Azure Arc-enabled server is using Private Link for its connection. Use your Azure Bastion session to run the command below:
```

I am trying to understand where this is coming from. also, if Bastion is required, how come it's a boolean in the ARM template and not a default _true_ value?

Reply:
My bad, let me change it